Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./bin/api
 
 # final stage
-FROM alpine:3.9
+FROM alpine:latest
```

Comment:
I would prefer to pin this to the current latest version

Reply:
I normally also would take fixed versions. With this case it's used solely for getting the ca-certificates. From my point of view it would be better to assure always having the newest ca-cert.

I can't see any breaking issues when not using the latest, therefor I thought it might be better to use latest.

What do you think?

Reply:
I normally also would take fixed versions. With this case it's used solely for getting the ca-certificates. From my point of view it would be better to assure always having the newest ca-cert.

I can't see any breaking issues when not using the latest, therefor I thought it might be better to use latest.

What do you think?

Reply:
I dare vote for pinning, if something changes and breaks a year from now it will be harder to find, especially if it only runs to get a cert :/ IMHO I'd prefer the inconvenience of updating the pinned version every once in a while than having it misbehave after some restart and looking for this thing. Also maybe we should pull the image from our own ACR (put it there and pull from there) in case something is changed by docker hub again?

Reply:
Alright, so I'll pin it.

Reply:
Done, is now 3.13